packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -576,6 +576,26 @@ export class FilesService {
         return [...baseUrl, file.id, filename].join("/");
     }
 
+    async createFileDownloadUrl(file: FileInterface, previewDamUrls?: boolean): Promise<string> {
+        const filename = parse(file.name).name;
+
+        // Use CDN url only if available and not in preview as preview requires auth
+        const baseUrl = [this.config.cdnEnabled ? `${this.config.cdnDomain}/files/download` : `${this.config.filesBaseUrl}/download`];
+
+        if (previewDamUrls) {
+            baseUrl.push("preview");
```

Comment:
Have you tried this? There's no `/download/preview/` endpoint, so I don't think this works.